### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by tkevans**
@runongirlrunon navigate bar is completely fixed. login & log out will display properly.

### CODE REVIEW

File: src/Template/Layout/default.ctp [hunk 1]
```diff
@@ -21,7 +21,7 @@ $cakeDescription = 'CakePHP: the rapid development php framework';
     <?= $this->Html->charset() ?>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>
-        <?= $cakeDescription ?>:
+        <?= $cakeDescription ?>
```
Comment (runongirlrunon) [inline src/Template/Layout/default.ctp]: this line can be taken out, no need for it. also the line where the variable is declared, too.

**Comment by runongirlrunon** [comment]
screen shot 2017-08-01 at 3 00 53 pm
Seeing this error when I run the server on your branch, can you fix please?

**Comment by runongirlrunon** [comment]
Better screenshot: 
screen shot 2017-08-01 at 3 03 47 pm

**Review by runongirlrunon — CHANGES_REQUESTED** [review]
Some comments: 
1. The deprecation error should be fixed
2. The menu bar is not responsive to the size of the window, please fix so that it resizes itself accordingly 
3. It'd be helpful to add some sort of border to the default layout CSS so that the body of the content doesn't touch the edges of the window.

**Comment by tkevans** [comment]
i fixed deprecation error, and when user log in the register button will disappear.
also I took out $cakeDescription

**Review by runongirlrunon — APPROVED** [review]

**Comment by runongirlrunon** [comment]
This closes https://github.com/summer17-csc648-team1/warehausmedia/issues/14.

**Pull request merged**